lib/backends/gcp-secrets-manager-backend.js

```diff
@@ -19,12 +19,16 @@ class GCPSecretsManagerBackend extends KVBackend {
    * @param {string} key - Key used to store secret property value in GCP Secrets Manager.
```

Comment:
can we add some comment 

```suggestion
   * @param {string} key - Key used to store secret property value in GCP Secrets Manager.
   * @param {boolean} keyOptions.isBinary - Is the secret base64 encoded? Set to true to handle as binary.
```

if you have a better suggestion Im game 😅